.github/workflows/CreateRelease.yml

```diff
@@ -155,29 +155,50 @@ jobs:
           just tar-headers
           just tar-static-lib
 
-      - name: Download benchmarks (Windows)
+      - name: Download benchmarks (Windows amd)
         uses: actions/download-artifact@v4
         with:
-          name: benchmarks_Windows_none
-          path: benchmarks_Windows_none
+          name: benchmarks_Windows_none_amd
+          path: benchmarks_Windows_none_amd
 
-      - name: Download benchmarks (Linux hyperv)
+      - name: Download benchmarks (Linux hyperv amd)
         uses: actions/download-artifact@v4
         with:
-          name: benchmarks_Linux_hyperv
-          path: benchmarks_Linux_hyperv
+          name: benchmarks_Linux_hyperv_amd
+          path: benchmarks_Linux_hyperv_amd
 
-      - name: Download benchmarks (Linux kvm)
+      - name: Download benchmarks (Linux kvm amd)
         uses: actions/download-artifact@v4
         with:
-          name: benchmarks_Linux_kvm
-          path: benchmarks_Linux_kvm
+          name: benchmarks_Linux_kvm_amd
+          path: benchmarks_Linux_kvm_amd
+
+      - name: Download benchmarks (Windows intel)
```

Comment:
Since there are a lot of this download-artifact action, maybe we could combine them into 1 using something like this? https://github.com/[actions/download-artifact](https://github.com/actions/download-artifact?tab=readme-ov-file#download-multiple-filtered-artifacts-to-the-same-directory)?tab=readme-ov-file#download-multiple-filtered-artifacts-to-the-same-directory

Reply:
Done ✅